Lib/test/test_capi/test_set.py

```diff
@@ -0,0 +1,175 @@
+import unittest
+
+from test.support import import_helper
+
+# Skip this test if the _testcapi module isn't available.
+_testcapi = import_helper.import_module('_testcapi')
+
+class set_child(set):
```

Comment:
Usually I call this a "subclass", not a "child". I'm more used to parent/child names when it comes to processes.